openlibrary/templates/covers/add.html

```diff
@@ -13,12 +13,12 @@
     $ action = doc.url('/add-cover')
     $ guideline = _("Cover Guidelines")
 
-<div class="popAlert" id="errors" style="display: $('block' if status else 'none')">
-$if status:
-    $if status.code == 1: $_("Please provide a valid image.")
-    $elif status.code == 2: $_("Please provide an image URL.")
-    $elif status.code == 3: $_("Please provide a valid image.")
-</div>
+<div class="popAlert" id="errors">
+    $if status:
+      $if status.code == 1: $_("Please provide a valid image.")
+      $elif status.code == 2: $_("Please provide an image URL.")
+      $elif status.code == 3: $_("Please provide a valid image.")
+  </div>
```

Comment:
```suggestion
$if status:
    $if status.code == 1: $_("Please provide a valid image.")
    $elif status.code == 2: $_("Please provide an image URL.")
    $elif status.code == 3: $_("Please provide a valid image.")
</div>
```
Revert indentation changes.